samples/Readme.md

```diff
@@ -0,0 +1,6 @@
+To help our partners get a head start on their connectors, we have provided several samples covering different types of connectors. These connectors are meant only as examples, and are provided as-is.
```

Comment:
The audience of the document is the partners.  Can we change this to "To help get a head start on connectors"?